go.mod

```diff
@@ -76,19 +76,19 @@ require (
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.30.3
-	k8s.io/apiextensions-apiserver v0.30.3
-	k8s.io/apimachinery v0.30.3
-	k8s.io/apiserver v0.30.3
+	k8s.io/api v0.31.1
```

Comment:
In my opinion, we should bump all Kubernetes modules to v0.31.1 rather than have a mix of v0.30 and v0.31 modules.

Reply:
There are still a few deps at v0.30.x, were they missed?